mqtt/forwarder.go

```diff
@@ -36,7 +37,7 @@ func NewForwarder(topic string, logger log.Logger) Forwarder {
 	}
 }
 
-func (f forwarder) Forward(id string, sub messaging.Subscriber, pub messaging.Publisher) error {
+func (f forwarder) Forward(ctx context.Context, id string, sub messaging.Subscriber, pub messaging.Publisher) error {
```

Comment:
We can replace the variable `ctx` with `_`, since it not used inside the function